src/plots/plots.js

```diff
@@ -1315,6 +1315,19 @@ plots.supplyLayoutGlobalDefaults = function(layoutIn, layoutOut, formatObj) {
         return Lib.coerce(layoutIn, layoutOut, plots.layoutAttributes, attr, dflt);
     }
 
+    // Special treatment for title.y that can be 'auto'
+    // or a number between 0 and 1.
+    function coerceTitleY() {
+        if(layoutIn.title && isNumeric(layoutIn.title.y)) {
+            var propOut = Lib.nestedProperty(layoutOut, 'title.y');
+            var opts = Lib.nestedProperty(plots.layoutAttributes, 'title.y').get();
```

Comment:
If we keep this, it doesn't need to be `nestedProperty` - `plots.layoutAttributes.title.y` would work, right?

Reply:
You're right. `Lib.valObjectMeta.number.coerceFunction` used to check the defined min/max bounds but the removal of min/max in the attribute definition rendered that obsolete (see above) :man_facepalming:

Reply:
See https://github.com/plotly/plotly.js/pull/3276/commits/5fb7b8f4aa180ee894dce63646510407fb535777